scripts/doc_deploy.sh

```diff
@@ -43,4 +43,10 @@ fi
 git checkout -q -b ${BRANCH_NAME}
 git add .
 git commit -q -m "Travis update from ${TRAVIS_BUILD_NUMBER}"
+
+if [ "${TRAVIS_PULL_REQUEST}" != "false" -o "${TRAVIS_BRANCH}" != "develop" ]; then
```

Comment:
Have you considered Travis' [conditional items](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/) or [script deployment](https://docs.travis-ci.com/user/deployment/script/) with [`on` conditions](https://docs.travis-ci.com/user/deployment#Conditional-Releases-with-on%3A)?

Reply:
I do want to run `doc_deploy.sh` every time (to make sure changes in deploy script is "tested" too)
What I don't need is that final push. So it easier for now to keep it as shell script with some conditionals,